tests/ut/language/parser/test_manual_scope_parsing.py

```
@@ -162,6 +162,24 @@ def main(self, x: pl.Tensor[[64], pl.FP32]) -> pl.Tensor[[64], pl.FP32]:
+        with pytest.raises(Exception, match="manual_dep_edges"):
+
+            @pl.program
+            class _Prog:
+                @pl.function(type=pl.FunctionType.InCore)
+                def k1(self, x: pl.Tensor[[64], pl.FP32]) -> pl.Tensor[[64], pl.FP32]:
+                    return x
+
+                @pl.function(type=pl.FunctionType.Orchestration)
+                def main(self, x: pl.Tensor[[64], pl.FP32]) -> pl.Tensor[[64], pl.FP32]:
+                    with pl.manual_scope():
+                        a, a_tid = pl.submit(self.k1, x)
```

Ruff (0.15.15)

[warning] Unpacked variable `a` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)